<launch>
    <node pkg="geometric_controller" type="geometric_controller_node" name="geometric_controller_node" output="screen">
        <rosparam command="load" file="$(find geometric_controller)/launch/controller_parameters.yaml" />      

    </node>

    <node pkg="trajectory_planning" type="trajectory_publisher_node" name="trajectory_publisher_node" output="screen" >
        <!--waypoints for trajectory generating-->
        <rosparam command="load" file="$(find geometric_controller)/launch/planner_parameters.yaml" />
    </node>

    <node pkg="rviz" type="rviz" name="rviz" output="screen"
    args="-d $(find geometric_controller)/config/traj.rviz" >
    </node>
</launch>